proposals/fh_discord.md

```diff
@@ -0,0 +1,47 @@
+
+## Community needs
+
+<!-- What specific issue are you addressing with your proposal? -->
+When it comes to determing whether or not Discord should be used over Slack, it's important to note that Slack isn't entirely free and has a data limit for messages and files for the free tier(That info can be found here https://get.slack.help/hc/en-us/articles/115002422943-Message-and-storage-limits-on-the-Free-plan). 
+
+With Discord, Florida Hackers will have the ability to grow its community without the need for limiting the amount of users in the chat. 
+
+Currently the Florida Hackers Slack channel is not as active as it used to be, and we have already surpassed our 10,000 message free limit meaning that older messages are getting deleted over time. We only have 1.2 GB left of file storage left. The last message made on our general chat was on the day May 30th.As of Wednesday, August 29th we only hit 2 daily active users with the all time high being attributed to August 22nd with 3 daily active users. 
+
+Surely the Florida Hackers community is more than just 2-3 daily active users, therefore I find it vital that we swtich to discord as a platform in order to grow a successful community! 
+
+## Resources
+
+<!-- What resources do you need to accomplish your proposal? -->
+Discord and support of FH
+<!-- How will you obtain those resources? -->
+With approvial of this proposal we can move forward
+
+Thanks Chris Rodriguez for this comment 
+
+"Discord provides it's own invitation management with the ability to create a permanent invite link. Should Florida Hackers choose to apply for partner status it would also allow for the server to have a branded link that acts as a permanent invite as well, something along the lines of https://discord.gg/floridahackers" 
+## Media
+
+<!-- How are you preparing to deliver your community impact project? -->
+Endorsing a Florida Hackers discord at every FL hackathon starting with ShellHacks could help grow the community exponentially
+<!-- What are you going to do to prepare to deliver the content? -->
+Create a pull request to change the florida hackers site to have a discord invite link
+## Timeline
+
+<!-- Develop a rough timeline for your propsal. -->
+Upon approval of proposal, so time is relative here
+
+1. 1 day to create and setup strucutre of discord
+2. 1 or 2 days afterwards to create pull request to change the florida hackers site to have the discord invite link
+3. After setup and creation, as well as the addition of the new changes to the site. Market the switch from Slack to Discord constantly
+   - Posting to the Florida Hackers facebook
+   - and have every individual hack club endorse the new discord
```

Comment:
Can we actually force clubs to do this? Individual clubs run their own Slack instances (UF Hackathoners still runs on Slack, I think) so forcing individual clubs to "endorse" the new Discord instance might be against their best interests 😬.